spinalcordtoolbox/math.py

```diff
@@ -13,10 +13,11 @@
 logger = logging.getLogger(__name__)
 
 
-def dilate(data, size, shape, dim=None):
+def morphomath(data, filter, size, shape, dim=None):
```

Comment:
I think this was better as two separate functions for two separate filters. Now the operation is implied by a string which makes it 2nd-class code.

The common parts can still be factored to share code (maybe `selem = _morpomath(...)`) while retaining a straightforward API.

Reply:
yes i hesitated between the current option and your suggestion, but your suggestion makes more sense-- will do! thanks

Reply:
done in 2886a99ef9a3acbe82d963ad1a3113449b7c60bf